### DISCUSSION TITLE
uses sealos 4.3 to install k8s error

### DISCUSSION THREAD

**Discussion by lizeyou0203** [labring/sealos#3733] [Q&A] [answered]
error message:

error execution phase preflight: [preflight] Some fatal errors occurred:
        [ERROR NumCPU]: the number of available CPUs 1 is less than the required 2
        [ERROR Mem]: the system RAM (960 MB) is less than the minimum 1700 MB
[preflight] If you know what you are doing, you can make a check non-fatal with `--ignore-preflight-errors=...`

The command executed is: sealos run labring/kubernetes:v1.23.17-4.3.0 labring/calico:v3.24.6 --masters 192.168.18.221,192.168.18.222,192.168.18.223 --nodes 192.168.18.224

what i want to ask is how to add parameter like `--ignore-preflight-errors=all` to `sealos run`

**Comment by bxy4543** [comment, marked as answer]
This resource limitation is unavoidable, please configure more than 2 CPUs for the machine

**Reply by lizeyou0203** [reply]
This is a virtual machine test environment, I just want to test the function, so the configuration is low, and there is no way to improve the configuration.

`kubeadm init` can use parameters `--ignore-preflight-errors=` skip error.

`sealos run` How to achieve a similar effect, if there is no way, you can only give up `sealos`...

**Reply by bxy4543** [reply]
The current runtime of sealos is only k8s, and it will support k3s and other runtimes in the future. Your test environment should be more suitable for using k3s

**Reply by lizeyou0203** [reply]
大佬,你应该也是用中文的吧.我只是想在虚拟机上测试sealos功能.虚拟机现在有性能不够.多台1核1G的.
如果你那边确认不能跳过这个报错的的话,那我就只能直接使用kubeadm了.
感谢大佬的回答!

**Reply by bxy4543** [reply]
是的，目前是没有跳过的功能的，因为这些值是不太适合跳过的

**Reply by lizeyou0203** [reply]
多谢大佬解惑!
